static/app/utils/fields/index.ts

```diff
@@ -1181,6 +1193,36 @@ const REPLAY_FIELD_DEFINITIONS: Record<ReplayFieldKey, FieldDefinition> = {
     kind: FieldKind.FIELD,
     valueType: FieldValueType.STRING,
   },
+  [ReplayFieldKey.CLICKED_CLASS]: {
+    desc: t('Classname of an element that was clicked'),
```

Comment:
I would either make these all more descriptors, or the exact attribute name (though textContent isn't really an attribute)

```suggestion
    desc: t('Class name of an element that was clicked'),
```

```suggestion
    desc: t('`class` of an element that was clicked'),
```

Reply:
went with `class` and updated some others to match.